shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/rule/ShardingRule.java

```diff
@@ -521,6 +521,16 @@ public DataNode getDataNode(final String logicTableName) {
         findBindingTableRule(logicTable).ifPresent(bindingTableRule -> result.putAll(bindingTableRule.getLogicAndActualTables(dataSourceName, logicTable, actualTable, availableLogicBindingTables)));
         return result;
     }
+
+    /**
+     * Get logic tables via actual table name.
+     *
+     * @param actualTable actual table name
+     * @return logic tables
+     */
+    public List<String> getLogicTablesByActualTable(final String actualTable) {
```

Comment:
Hi @NOLouser, please modify the return type to collection, and add unit test for this method.

Reply:
@strongduanmu 
Sorry for the late reply, I will improve these later.

Reply:
> 

@NOLouser Take it easy and arrange it according to your time.